src/migrations/20210105154127-users-home-region-allow-null.js

```diff
@@ -1,15 +1,9 @@
 module.exports = {
-  up: async (queryInterface, Sequelize) => {
```

Comment:
Can we remove this file? Or should we remove the change in `add-title-to-user` and uncomment these lines?